### PR TITLE
Fixing the linebreak-style eslint error with version control systems's rule

### LINKED ISSUE #649: Disabling the linebreak-style rule in .eslintrc.js
## Description
On a windows machine, there's a ton of eslint errors when you run "npm run dev" about "expecting line breaks LF instead of CRLF". I've added
 *.js text eol=lf 
to my gitattributes file, according to eslint documentation (https://eslint.org/docs/rules/linebreak-style) but its not still not working.

## To Reproduce
1. Fork the repo
2. Clone it on your windows machine
3. Run 'npm install'
4. Run 'npm run dev' and you'll see the error

## Desktop (please complete the following information)
 - OS: (windows 10)
 - Browser (ex. chrome, safari)

### PR DESCRIPTION
**Pull request by abhisheknaiidu**
Fixes #649 

## Description
Version control systems sometimes have special behavior for linebreaks. For example, the default behavior of git on Windows systems is to convert LF linebreaks to CRLF when checking out files, but to store the linebreaks as LF when committing a change.

To make it easy for developers to contribute to the codebase from different platforms, we should configure VCS to handle linebreaks appropriately.

If using git, one option is to add a line to the `.gitattributes` file to prevent git from converting linebreaks in .js files:

`*.js text eol=lf`

That works with the default `"unix"` option for the linebreak-style rule.

Another option is to explicitly ask git to use native linebreaks for all users, regardless of their personal settings, by adding the following line to your `.gitattributes` file:

`* text=auto`

Source : https://eslint.org/docs/rules/linebreak-style#using-this-rule-with-version-control-systems

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,3 +17,4 @@ selenium-debug.log
 *.ntvs*
 *.njsproj
 *.sln
+*.js text eol=lf
```
Comment (brenoferreira) [inline .gitignore:20]: This should be included in the  `.gitattributes file`, not `.gitignore`

Comment (abhisheknaiidu) [reply .gitignore]: oops so sorry, my bad! Yea now I have added it in `.gitattributes file` 
Do you need to add some other formats also like 
`*.jsx   eol=lf`
`*.json  eol=lf`
@brenoferreira

**Comment by brenoferreira** [comment]
We've make some updates recently to the webpack and npm run dev doesn't show lint errors anymore. I think this is no longer relevant.

**Comment by brenoferreira** [comment]
Thanks for your contribution.

**Pull request merged**